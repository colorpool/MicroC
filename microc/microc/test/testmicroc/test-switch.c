void main(int n){
  switch(n){
    case 1: {print 'd' : 1; break;}
    case 2: {print 'd' : 0; break;}
    default: print 'd' : 233;
  }
  print 'd' : n;
}