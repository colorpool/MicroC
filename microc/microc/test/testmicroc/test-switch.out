24 19 1 5 25 13 0 0 1 11 0 1 6 17 26 0 1 22 15 -1 16 67 15 0 16 36 13 0 0 1 11 0 2 6 17 47 0 0 22 15 -1 16 67 15 0 16 60 13 0 0 1 11 13 0 0 1 11 6 17 67 0 233 22 15 -1 16 67 13 0 0 1 11 22 15 -1 15 0 21 0